packages/suite-desktop-core/e2e/support/pageActions/settingsActions.ts

```diff
@@ -4,7 +4,7 @@ import { BackendType, NetworkSymbol } from '@suite-common/wallet-config';
 
 export class SettingsActions {
     private readonly window: Page;
-    private readonly TIMES_CLICK_TO_SET_DEBUG_MODE = 5;
+    private readonly TIMES_CLICK_TO_SET_DEBUG_MODE = 7;
```

Comment:
this has changed?

Reply:
It is still 5.
But I have encountered a test run where 5 clicks failed. My assumption is the page probably was not fully loaded and the counter got reset once it finished. So either we find the event that did that and try to check for loaded page or we can keep a this extra buffer of clicks.